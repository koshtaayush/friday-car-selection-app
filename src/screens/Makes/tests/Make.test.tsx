import React from 'react'
import { shallow } from 'enzyme'
import Make from '..'

jest.mock('react-router-dom', () => ({
    ...jest.requireActual('react-router-dom'),
    useHistory: () => ({
        push: jest.fn(),
    }),
}))

describe('Make', () => {

    it('Should match the snapshot', () => {
        const component = shallow(
            <Make />
        )

        expect(component).toMatchSnapshot()
    })
})
