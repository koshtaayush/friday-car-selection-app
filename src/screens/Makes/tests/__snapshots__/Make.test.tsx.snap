// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Make Should match the snapshot 1`] = `
<Fragment>
  <Search
    onChangeProp={[Function]}
    placeholder="Search for a make"
    value=""
  />
  <styled.div />
</Fragment>
`;
